dashboard/agent.py

```diff
@@ -24,6 +24,11 @@
 from ray.core.generated import agent_manager_pb2_grpc
 import psutil
 
+try:
+    create_task = asyncio.create_task
+except AttributeError:
+    create_task = asyncio.ensure_future
```

Comment:
Yes, the `asyncio.ensure_future` is the older version of `asyncio.create_task`. From the official doc of `asyncio.create_task` https://docs.python.org/3/library/asyncio-task.html#asyncio.create_task:
```
This function has been added in Python 3.7. Prior to Python 3.7,
the low-level asyncio.ensure_future() function can be used instead
```

Reply:
Why not just always using ensure_future in this case?

Reply:
Because Python doc recommends using `asyncio.create_task`. see: https://docs.python.org/3/library/asyncio-future.html#asyncio.ensure_future
```
Important See also the create_task() function which is the preferred way for creating new Tasks.
```